<launch>
  <arg name="base_path" default="/home/helen/data/jfr_2018/shed/voxblox/" />
  <arg name="esdf_name" default="stereo_esdf_0.10.voxblox" />
  <arg name="sparse_graph_name" default="stereo_sparse_graph_0.10.voxblox" />
  <arg name="results_name" default="stereo_results_12_10_2018c.csv" />

  <arg name="frame_id" default="map" />

  <node name="global_planning_benchmark" pkg="mav_planning_benchmark" type="global_planning_benchmark_node" output="screen">
    <param name="base_path" value="$(arg base_path)" />
    <param name="esdf_name" value="$(arg esdf_name)" />
    <param name="sparse_graph_name" value="$(arg sparse_graph_name)" />
    <param name="results_name" value="$(arg results_name)" />

    <param name="visualize" value="true" />
    <param name="robot_radius" value="0.5" />
    <param name="tsdf_voxel_size" value="0.10" />
    <param name="tsdf_voxels_per_side" value="16" />
    <param name="update_mesh_every_n_sec" value="0.0" />
    <param name="num_seconds_to_plan" value="5.0" />
    <param name="frame_id" value="$(arg frame_id)" />
    <param name="world_frame" value="$(arg frame_id)" />
    <param name="verbose" value="true" />
  </node>

</launch>
